src/main/kotlin/com/atlassian/performance/tools/jiraactions/page/IssueCreateDialog.kt

```diff
@@ -49,6 +51,26 @@ internal class IssueCreateDialog(
         return this
     }
 
+    /**
+     * Click 'Configure Fields' to display 'All' to ensure
+     * all mandatory fields are displayed in creation dialog.
+     *
+     */
+    fun showAllFields(): IssueCreateDialog {
+        driver.wait(elementToBeClickable(configColumnField)).click()
+        val configureFieldsDialogId = "inline-dialog-field_picker_popup"
+        driver.wait(visibilityOfElementLocated(By.id(configureFieldsDialogId)))
+        val locator = By.xpath("//div[@id='$configureFieldsDialogId']//dd[1]//a")
```

Comment:
@wyrzyk IMHO this looks a lot less stable than before

Reply:
@wyrzyk ping
We need the PR, so we need to reach a consensus here. If you believe this is better than `text()='All'`, then I'm fine with that.

Reply:
IMO both approaches are not stable. Hard to say which one is better. It's not a very critical change, we can fix it later if it'll be problematic.

Reply:
sure.
Agree that neither of the approaches is stable, I may spend a bit more time later to see if there is any alternatives.